content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,271 @@
+
+```sh
+$ influx config ls
+Active	Name		URL					Org
+ 	default		http://localhost:8086			InfluxData
+*	influx_old	http://localhost:9999			InfluxData
+```
+
+Now you can send commands to each of them as needed using the `-c, --active-config` option on the CLI.
+
+## 7. Copy all resources from old instance to the new one
+
+Now we can copy all your existing influxdb resources, such as dashboards, tasks, and alerts to your new instance using a single command.
+
+```sh
+influx export all -c influx_old | influx apply -c default
+```
+
+Check out our documentation for the [`influx export`](/influxdb/v2.0/reference/cli/influx/export/)
+and [`influx apply`](/influxdb/v2.0/reference/cli/influx/apply/) commands if you'd like more details about what this is doing.
+
+You should see a list of the resources being created in your new instance.
+Assuming everything went ok, you can feel free to delete the bucket created during the setup.
+
+```sh
+$ influx export all -c influx_old | influx apply -c default
+LABELS    +add | -remove | unchanged
++-----+------------------------+----+---------------+---------+-------------+
+| +/- |     METADATA NAME      | ID | RESOURCE NAME |  COLOR  | DESCRIPTION |
++-----+------------------------+----+---------------+---------+-------------+
+| +   | tasty-northcutt-c9c001 |    | something     | #326BBA |             |
++-----+------------------------+----+---------------+---------+-------------+
+|                                                      TOTAL  |      1      |
++-----+------------------------+----+---------------+---------+-------------+
+​
+BUCKETS    +add | -remove | unchanged
++-----+------------------------+----+---------------+------------------+-------------+
+| +/- |     METADATA NAME      | ID | RESOURCE NAME | RETENTION PERIOD | DESCRIPTION |
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | fasting-taussig-c9c007 |    | apps          | 0s               |             |
++-----+------------------------+----+---------------+------------------+-------------+
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | great-davinci-c9c005   |    | new_telegraf  | 0s               |             |
++-----+------------------------+----+---------------+------------------+-------------+
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | stubborn-hugle-c9c003  |    | telegraf      | 719h59m59s       |             |
++-----+------------------------+----+---------------+------------------+-------------+
+|                                                          TOTAL       |      3      |
++-----+------------------------+----+---------------+------------------+-------------+
+```
+
+Now you have all the resources from your old instance stored in your new instance.
+Log into your new instance (probably http://localhost:8086) and take a look.
```

Comment:
We haven't touched your data yet, so your dashboards will probably look a little empty.